I get the following error when I run the test using pytest in breeze environment for Elasticsearch.  Could you share your insights on where I am going wrong? I have configured to run the ES in local env and updated the username and password for `elasticsearch_default` conn_id
@potiuk 
```
root@20f758b954e9:/opt/airflow/tests/system/providers/elasticsearch# pytest --system elasticsearch example_elasticsearch_query.py
============================= test session starts ==============================
platform linux -- Python 3.7.13, pytest-6.2.5, py-1.11.0, pluggy-1.0.0 -- /usr/local/bin/python
cachedir: .pytest_cache
rootdir: /opt/airflow, configfile: pytest.ini
plugins: timeouts-1.2.1, httpx-0.20.0, asyncio-0.18.3, instafail-0.4.2, anyio-3.5.0, requests-mock-1.9.3, cov-3.0.0, flaky-3.7.0, rerunfailures-9.1.1, forked-1.4.0, xdist-2.5.0
asyncio: mode=strict
setup timeout: 0.0s, execution timeout: 0.0s, teardown timeout: 0.0s
collecting ... collected 1 item

example_elasticsearch_query.py::test_run <- tests/system/utils/__init__.py FAILED [100%]

=================================== FAILURES ===================================
___________________________________ test_run ___________________________________
ValueError: None is not a valid DagRunState

During handling of the above exception, another exception occurred:

    def test_run():
>       dag.clear(dag_run_state=State.NONE)

../../utils/__init__.py:22:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
../../../../airflow/utils/session.py:71: in wrapper
    return func(*args, session=session, **kwargs)
../../../../airflow/models/dag.py:1856: in clear
    dag_run_state=dag_run_state,
../../../../airflow/models/taskinstance.py:289: in clear_task_instances
    dag_run_state = DagRunState(dag_run_state)  # Validate the state value.
/usr/local/lib/python3.7/enum.py:315: in __call__
    return cls.__new__(cls, value)
/usr/local/lib/python3.7/enum.py:569: in __new__
    raise exc
/usr/local/lib/python3.7/enum.py:553: in __new__
    result = cls._missing_(value)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

cls = <enum 'DagRunState'>, value = None

    @classmethod
    def _missing_(cls, value):
>       raise ValueError("%r is not a valid %s" % (value, cls.__name__))
E       ValueError: None is not a valid DagRunState

/usr/local/lib/python3.7/enum.py:582: ValueError
---------------------------- Captured stdout setup -----------------------------
========================= AIRFLOW ==========================
Home of the user: /root
Airflow home /root/airflow
Skipping initializing of the DB as it was initialized already.
You can re-initialize the database by adding --with-db-init flag when running tests.
=============================== warnings summary ===============================
../../../../airflow/configuration.py:398
  /opt/airflow/airflow/configuration.py:398: FutureWarning: The 'dag_default_view' setting in [webserver] has the old default value of 'tree'. This value has been changed to 'grid' in the running config, but please update your config before Apache Airflow 3.0.
    FutureWarning,

../../../../airflow/configuration.py:398
  /opt/airflow/airflow/configuration.py:398: FutureWarning: The 'log_filename_template' setting in [logging] has the old default value of '{{ ti.dag_id }}/{{ ti.task_id }}/{{ ts }}/{{ try_number }}.log'. This value has been changed to 'dag_id={{ ti.dag_id }}/run_id={{ ti.run_id }}/task_id={{ ti.task_id }}/{%% if ti.map_index >= 0 %%}map_index={{ ti.map_index }}/{%% endif %%}attempt={{ try_number }}.log' in the running config, but please update your config before Apache Airflow 3.0.
    FutureWarning,

-- Docs: https://docs.pytest.org/en/stable/warnings.html
=========================== short test summary info ============================
FAILED example_elasticsearch_query.py::test_run - ValueError: None is not a v...
======================== 1 failed, 2 warnings in 1.15s =========================
```

--- a/docs/apache-airflow-providers-elasticsearch/connections/elasticsearch.rst
+++ b/docs/apache-airflow-providers-elasticsearch/connections/elasticsearch.rst
@@ -71,3 +71,9 @@ For example:
 .. code-block:: bash
 
     export AIRFLOW_CONN_ELASTICSEARCH_DEFAULT='elasticsearch://elasticsearchlogin:elasticsearchpassword@elastic.co:80/http'
+
+.. exampleinclude:: /../../tests/system/providers/elasticsearch/example_elasticsearch_query.py
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_elasticsearch_query]
+    :end-before: [END howto_elasticsearch_query]

--- a/tests/system/providers/elasticsearch/example_elasticsearch_query.py
+++ b/tests/system/providers/elasticsearch/example_elasticsearch_query.py
@@ -1,0 +1,66 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG for Elasticsearch Query.
+"""
+import os
+from datetime import datetime
+
+from airflow import models
+from airflow.decorators import task
+from airflow.providers.elasticsearch.hooks.elasticsearch import ElasticsearchHook
+
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+DAG_ID = 'elasticsearch_dag'
+CONN_ID = 'elasticsearch_default'
+
+
+@task(task_id='es_print_tables')
+def show_tables():
+    """
+    show_tables queries elasticsearch to list available tables
+    """
+    # [START howto_elasticsearch_query]
+    es = ElasticsearchHook(elasticsearch_conn_id=CONN_ID)
+
+    # Handle ES conn with context manager
+    with es.get_conn() as es_conn:
+        tables = es_conn.execute('SHOW TABLES')
+        for table, *_ in tables:
+            print(f"table: {table}")
+    return True
+    # [END howto_elasticsearch_query]
+
+
+with models.DAG(
+    DAG_ID,
+    schedule_interval="@once",
+    start_date=datetime(2021, 1, 1),
+    catchup=False,
+    tags=["example", "elasticsearch"],
+) as dag:
+    execute_query = show_tables()
+    (
+        # TEST BODY
+        execute_query
+    )
+
+from tests.system.utils import get_test_run  # noqa: E402
+
+# Needed to run the example DAG with pytest (see: tests/system/README.md#run_via_pytest)
+test_run = get_test_run(dag)

--- a/tests/system/utils/__init__.py
+++ b/tests/system/utils/__init__.py
@@ -19,7 +19,7 @@ from airflow.utils.state import State
 
 def get_test_run(dag):
     def test_run():
-        dag.clear(dag_run_state=State.NONE)
+        dag.clear(dag_run_state=State.QUEUED)
         dag.run()
 
     return test_run

